src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -250,6 +254,7 @@ export class ExtensionEditor extends BaseEditor {
 		this.navbar.onChange(this.onNavbarChange.bind(this, extension), this, this.transientDisposables);
 		this.navbar.push(NavbarSection.Readme, localize('details', "Details"));
 		this.navbar.push(NavbarSection.Contributions, localize('contributions', "Contributions"));
+		this.navbar.push(NavbarSection.Changelog, localize('changelog', "Changelog"));
```

Comment:
I'd like to show the Changelog section only if there is actually a Changelog to show.

You can add a `hasChangelog(): boolean` method to `IExtension` that will let you know this synchronously.